video_store.py

```diff
@@ -0,0 +1,121 @@
+import requests
+import datetime
+from flask import json
```

Comment:
since  this is a front-end CLI that simply interacts with the user in the terminal, we don't need to bring in `flask`, which is a backend framework. the `.json()` method is part of the `requests` module we've imported

```suggestion

```